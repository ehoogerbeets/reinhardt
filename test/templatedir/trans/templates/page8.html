{% blocktrans key "unique_key_block" %}
This is a blocktrans block with a unique key.
{% endblocktrans %}
