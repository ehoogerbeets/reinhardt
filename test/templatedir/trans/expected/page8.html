 Dies ist eine blocktrans Block mit einem eindeutigen SchlÃ¼ssel. 
